src/plugins/intel_gna/src/layers/gna_layer_info.hpp

```diff
@@ -62,15 +62,15 @@ class LayerInfo {
     // The name of the funciton may be somehwat misleading
     // Explanation: when in low precision mode the listed layers have 8-bit outputs
     // and when in 16-bit input mode, they have 16-bit outputs
-    bool has8BOr16BOutput() const noexcept {
```

Comment:
Are you sure that the right fix to allow throw exceptions in this function? 
IMO this function should return true or false and should not throw any exceptions.

Reply:
I guess this is necessery if don't want to change rest of the code
there is used isCropAffined() which calls GetCropParams() which calls get_crop_params)_ this is may throw.
if we don't want to have excetion in this methods we need to redesign this methods to return error information as argument and handle this one level upper.

currently this method can throw but noexcept suggest the it shouldn't.

Reply:
I'm ok with that solution, because this code is legacy.